Zend/zend_ast.c

```diff
@@ -2079,6 +2084,12 @@ static ZEND_COLD void zend_ast_export_ex(smart_str *str, zend_ast *ast, int prio
 			ast = ast->child[1];
 			goto tail_call;
 
+		case ZEND_AST_INITIALIZER_EXPR:
+			smart_str_append(str, zend_ast_get_str(ast->child[0]));
+			smart_str_appends(str, "= ");
```

Comment:
Can merge with ZEND_AST_NAMED_ARG

```suggestion
			smart_str_appends(str, ": ");
```

Reply:
Thanks for spotting this! I originally used `=` in the initializer expression syntax, so this one stuck here from that time.